core/arch/arm/plat-imx/crypto_conf.mk

```diff
@@ -0,0 +1,69 @@
+#
+# Define the cryptographic algorithm to be built
+#
+
+#
+# CAAM Debug Trace
+#
+# DBG_TRACE_HAL  BIT32(0)  // HAL trace
+# DBG_TRACE_CTRL BIT32(1)  // Controller trace
+# DBG_TRACE_MEM  BIT32(2)  // Memory utility trace
+# DBG_TRACE_PWR  BIT32(3)  // Power trace
+# DBG_TRACE_JR   BIT32(4)  // Job Ring trace
+# DBG_DESC_JR    BIT32(5)  // Job Ring dump descriptor
+# DBG_TRACE_RNG  BIT32(6)  // RNG trace
+# DBG_DESC_RNG   BIT32(7)  // RNG dump descriptor
+# DBG_TRACE_HASH BIT32(8)  // Hash trace
+# DBG_DESC_HASH  BIT32(9)  // Hash dump descriptor
+# DBG_BUF_HASH   BIT32(10) // Hash dump Buffer
+CFG_CAAM_DBG ?= 0x2
+
+#
+# CAAM Job Ring configuration
+#  - Normal boot settings
+#  - HAB support boot settings
+#
+$(call force, CFG_JR_BLOCK_SIZE,0x1000)
+
+$(call force, CFG_JR_INDEX,0)  # Default JR index used
+$(call force, CFG_JR_IRQ,137)  # Default JR IT Number (105 + 32) = 137
```

Comment:
These can probably be removed and carried as regular defines until support for another platform is added.

Reply:
It's not possible because we already support multi-platforms (like LS)